### ISSUE TITLE
Error when scanning fastmail hosted SMTP

### ISSUE THREAD

**Issue by davidillsley** [hardenize/hardenize-public#2]
It seems that fastmail's server is blacklisting your scanner for lots of abnormal behaviour. This shows up as a failure "We've detected serious problems that require your immediate attention." despite the servers actually supporting STARTTLS. In the text from the 'Operational' column it says:
`< 421 Client host [23.253.156.165] blocked using internal list; Too many timeouts. Expected removal in 3.2 days`

Given the large number of domains fastmail hosts email for, this probably affects lots of domains (it does one of mine). Maybe it's possible for them to whitelist you? Of for you to scan them once-per server, rather than once per domain? Or for you to whitelist them?

**Comment by ivanr** [comment]
Hi @davidillsley. Yes, I've noticed. I've made some changes to how this problem is presented. It's no longer an error and there is now an explicit warning attached to the email section: https://www.hardenize.com/report/scotthelme.co.uk/1504786356#email I will probably introduce a new test status ("inconclusive" or something similar) because "all ok" is not appropriate either.

Going forward, I will introduce a special partial assessment cache so that I avoid hitting the popular mail servers too often, have a set of static outbound addresses, and then engage the mail providers to see and if we can get along.
